MonitoringSystem/Client/Types/BaseType.py

```diff
@@ -146,4 +151,30 @@ def addMapping(self, mapping):
 
   ########################################################################
   def getMapping(self):
+    """
+    It returns a specific mapping, which is used by a certain monitoring type 
+    """
     return self.__mapping
+
+  ########################################################################
+  def getPeriod( self ):
+    """
+    
+    It returns the indexing period.
+    
+    """
+    return self.__period
```

Comment:
I rest of my idea that we don't need these setters/getters, and if needed we can use properties, which is a more pythonic way to do things. Anyway, this comment can be ignored for this specific PR...